src/editors/sharedComponents/TypeaheadDropdown/index.jsx

```diff
@@ -90,7 +90,7 @@ class TypeaheadDropdown extends React.Component {
       this.setValue(opt);
       this.setState({ displayValue: opt });
     } else {
-      this.setValue(normalized);
+      this.setValue(value);
```

Comment:
As it's done in L90 for existent orgs, saving the `value` instead looks like the right thing to do for newly created. Thank you!